server/plugins/LFUGens.cpp

```diff
@@ -2817,7 +2817,7 @@ static inline bool check_gate_ar(EnvGen * unit, int i, float & prevGate, float *
 
 static inline bool EnvGen_nextSegment(EnvGen * unit, int & counter, double & level)
 {
-	//if (unit->m_stage == ENVGEN_NOT_STARTED) { return true; } // this fixes doneAction 14, but breaks with EnvGen_next_aa
+	if (unit->m_stage == ENVGEN_NOT_STARTED) { counter = INT_MAX; return true; } 
```

Comment:
please split this block out so the statements are on separate lines

Reply:
This needs to be done in the source branch first. This isn't my commit.

Reply:
Or... I guess I could do it here. We merge #3671 first, then this one, and if I've added a commit to fix the formatting, then it should be ok.

Still, this change is originally from #3671. More effective to mention it to the original author.

Reply:
good point, I've left a comment there.

Reply:
Hm, now that I think about it, I wonder if that's going to cause a git workflow problem.

Currently, this branch is based on #3671 as is. So, if you merge #3671 and then this one, no problem.

I'm not sure what will happen if #3671 changes.

Maybe I should redo this PR to include only the test suite.